webdriver-spec.html

```diff
@@ -3884,6 +3884,16 @@
 
  <li><p>Let <var>selector</var> be equal to <var>value</var>.
 
+ <li><p>If <var>selector</var> is equal to <code>""</code> or <a>null</a>, match on <var>all</var>:
```

Comment:
This is currently what Selenium does, It would allow us to return earlier than having to pass this off to `querySelector` to do the work.